app/src/main/kotlin/com/mapzen/erasermap/view/MainActivity.kt

```diff
@@ -455,11 +476,11 @@ public class MainActivity : AppCompatActivity(), MainViewController, RouteCallba
 
     override fun success(route: Route) {
         routeManager?.route = route
-        routePreviewView?.route = route
```

Comment:
Indeed. If the `findViewById(...)` lookup fails to find the expected type (in the lazy initializer above) the app will crash. It also means something is seriously wrong so crashing is probably the best option available at that point.